CODE_OF_CONDUCT.md

```diff
@@ -169,18 +155,18 @@ Upon receiving a report of an incident, the ELIXIR Code of Conduct Oversight Com
 - who, if anyone, was the bad actor
 
 This information will be collected either in person or in writing.
-The ELIXIR Code of Conduct Oversight Committee  will provide a written summary of the information surrounding the incident.
+The RDM CoC group will provide a written summary of the information surrounding the incident.
 All participants will be anonymised in the summary report, referred to as "Community Member 1", "Community Member 2", or "Research Team Member 1".
 The "de-anonymising key" will be kept in a separate file and only accessed to link repeated reports against the same person over time.
 
-The ELIXIR Code of Conduct Oversight Committee  will aim to have a resolution agreed upon within one week.
-In the event that a resolution can't be determined in that time, a member of the CoC Oversight committee will respond to the reporter(s) with an update and projected timeline for resolution.
+The RDM CoC group will aim to have a resolution agreed upon within one week.
+In the event that a resolution can't be determined in that time, a member of the RDM CoC group will respond to the reporter(s) with an update and projected timeline for resolution.
 
 ### 4.4 Resolutions
 
-The ELIXIR Code of Conduct Oversight Committee  will seek to agree on a resolution by consensus of all members investigating the report in question.
-If the committee cannot reach consensus and deadlocks for over a week, Niklas Blomberg, as Director of ELIXIR, will break the tie.
-If Dr Blomberg is unable to take part in the discussion due to a conflict of interest, Susanna Rep, as an external member of the ELIXIR Code of Conduct Oversight committee, will make the decision.
+The RDM CoC group will seek to agree on a resolution by consensus of all members investigating the report in question.
+If the committee cannot reach consensus in over a week, Niklas Blomberg, as Director of ELIXIR, will descide on an appropriate resolution.
```

Comment:
```suggestion
If the committee cannot reach consensus within a week, Niklas Blomberg, as Director of ELIXIR, will decide on an appropriate resolution.
```

Reply:
This comment is still standing.

Reply:
@nsoranzo I was a bit to quick merging this. I updated it in the master branch. Thank you for checking out on this!